source/release-notes.txt

```diff
@@ -10,6 +10,37 @@ Release Notes
    :depth: 1
    :class: twocols
 
+|compass| 1.37.0
+----------------
+
+*Released May 25, 2023*
+
+New Features:
+
+- Enable "Modern Dark Mode" by default
+- Surface type warnings (:issue:`COMPASS-6543`)
```

Comment:
Where do these type warnings appear? Can we be more specific about how this impacts users?